book/modules.md

```diff
@@ -147,73 +145,45 @@ hi there!
 ## Environment Variables
 
 So far we used modules just to import custom commands.
-It is possible to export environment variables the same way.
-The syntax is slightly different than what you might be used to from commands like [`let-env`](commands/let-env.md) or [`load-env`](commands/load-env.md):
+However, modules can also define an environment using [`export-env`](commands/export-env.md):
 
 ```
 # greetings.nu
 
-export env MYNAME { "Arthur, King of the Britons" }
+export-env {
+    let-env MYNAME = "Arthur, King of the Britons"
+}
 
-export def hello [name: string] {
-    $"hello ($name)"
+export def hello [] {
+    $"hello ($env.MYNAME)"
 }
 ```
 
-`use` works the same way as with custom commands:
-
+`use` will run the code inside the `export-env` block and merge its environment into the current scope:
```

Comment:
Users will wonder why they can't simply use `let-env` at the top-level in their module, since the feature seems to behave exactly as one would expect with a top-level `let-env` (i.e. the variables are available in `def` bodies inside the module, and the variables get merged into the current scope when doing `use`). Do you think it's worth adding a sentence explaining why it's not done using a top-level `let-env` in a module?

Reply:
Well, it's not just about `let-env` but also `load-env` and any other calls that might bring in environments (e.g., defined with `def-env`) and in fact basically any imaginable code, including variables, that you might need for setting up the environment. Inside `export-env` it's nicely scoped and won't pollute the namespace of other commands inside the module. I'll add a sentence about this.